deploy/runtools/runtime_config.py

```diff
@@ -198,23 +218,29 @@ def build_fpga_driver(self) -> None:
         design = triplet_pieces[0]
         target_config = triplet_pieces[1]
         platform_config = triplet_pieces[2]
-        rootLogger.info("Building FPGA software driver for " + str(self.get_deploytriplet_for_config()))
+        rootLogger.info("Building " + self.driver_type_message + " driver for " + str(self.get_deploytriplet_for_config()))
         with prefix('cd ../'), \
              prefix('export RISCV={}'.format(os.getenv('RISCV', ""))), \
              prefix('export PATH={}'.format(os.getenv('PATH', ""))), \
              prefix('export LD_LIBRARY_PATH={}'.format(os.getenv('LD_LIBRARY_PATH', ""))), \
              prefix('source ./sourceme-f1-manager.sh'), \
              prefix('cd sim/'), \
              StreamLogger('stdout'), \
-             StreamLogger('stderr'):
+             StreamLogger('stderr'), \
+             prefix('set -o pipefail'):
             localcap = None
             with settings(warn_only=True):
-                driverbuildcommand = """make DESIGN={} TARGET_CONFIG={} PLATFORM_CONFIG={} PLATFORM={} driver""".format(design, target_config, platform_config, self.platform)
-                localcap = local(driverbuildcommand, capture=True)
-            rootLogger.debug("[localhost] " + str(localcap))
-            rootLogger.debug("[localhost] " + str(localcap.stderr))
+                # the local driver dir must already exist for the tee to always
+                # work
+                local("""mkdir -p {}""".format(self.get_local_driver_dir()))
+                buildlogfile = """{}firesim-manager-make-{}-temp-output-log""".format(self.get_local_driver_dir(), self.driver_build_target)
+                driverbuildcommand = """make DESIGN={} TARGET_CONFIG={} PLATFORM_CONFIG={} {}""" .format(design, target_config, platform_config, self.driver_build_target)
```

Comment:
where did `PLATFORM={}` go? That's critical for local fpga support.

Reply:
fixed